playhouse/tests/test_fields.py

```diff
@@ -453,7 +453,7 @@ def test_timestamp_field(self):
 
         expected = datetime.datetime(2016, 1, 3, 12, 12, 13)
         self.assertEqual(t3_db.local_us, expected.replace(microsecond=654321))
-        self.assertEqual(t3_db.utc_ms, expected.replace(microsecond=654))
+        self.assertEqual(t3_db.utc_ms, expected.replace(microsecond=654000))
```

Comment:
Ouch, my assertions were straight up testing the wrong value. Regret I didn't think it through a bit more, and apologies if this caused you all any problems.